### PR TITLE
Additional logging required for sec reqs

### PR DESCRIPTION
**Pull request by bmallred**

### CODE REVIEW

**Comment by floresj** [comment]
Captures additional request information including user account id, ip and method type.
selection_646

File: api/db/db.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ func NewDB() *DatabaseContext {
 			log.WithFields(logrus.Fields{
 				"elapsed": time.Since(event.StartTime),
 				"query":   query,
-			}).Debug("Excuted database query")
+			}).Debug("Executed database query")
```
Comment (floresj) [inline api/db/db.go]: X-eh-queue-ted

File: api/jwt/token.go [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+)
+
+var (
+	JwtSecret        = []byte(os.Getenv("JWT_SECRET"))
+	AuthBearerRegexp = regexp.MustCompile("Bearer\\s(.*)")
+	Expiration       = time.Hour * 1
+)
+
+func CheckToken(r *http.Request, validTokenFunc func(string, string) (bool, error), audiences ...string) (string, error) {
+	// log := logmsg.NewLogger(r)
```
Comment (floresj) [inline api/jwt/token.go]: Were ya going to add log statements these back in?

Comment (bmallred) [reply api/jwt/token.go]: Negative due to cyclical imports. I'll remove those!

**Pull request merged**